### PR TITLE
Update "broccoli-uglify-sourcemap" to v2.0.0-beta.1

### PR DESCRIPTION
**Pull request by Turbo87**
related to https://github.com/ember-cli/broccoli-uglify-sourcemap/pull/49

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,53 @@
...
 'use strict';
 
 module.exports = {
   name: 'ember-cli-uglify',
+
   included: function(app) {
     this._super.included.apply(this,arguments);
 
-    this.options = app.options;
+    const defaults = require('lodash.defaultsdeep');
```
Comment (rwjblue) [inline index.js:9]: This is not included in `package.json`

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Turbo87) [reply index.js:9]: whoops, fixed

**Review by rwjblue — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -23,6 +23,9 @@
     "before": "broccoli-asset-rev"
   },
   "dependencies": {
-    "broccoli-uglify-sourcemap": "^1.0.0"
+    "broccoli-uglify-sourcemap": "^2.0.0-beta.1"
+  },
+  "devDependencies": {
+    "lodash.defaultsdeep": "^4.6.0"
```
Comment (rwjblue) [inline package.json]: Wait, this should be a dep not a devDep

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Turbo87) [reply package.json]: I should go to bed... fixed

**Comment by Turbo87** [comment]
since the two comments are addressed I'm merging and releasing as beta too

**Pull request merged**